src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -95,10 +95,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + feedbackResponseComment.feedbackSessionName + "<br>"
                            + "by: " + feedbackResponseComment.giverEmail + "<br>"
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
+            // createdAt, lastEditedAt, commentGiverName and commentEditorName are required to generate editedCommentDetails
```

Comment:
You can take a look at the other `*Action` classes and use whichever style seems appropriate. From a quick survey, it seems that using `//` on multiple lines is more common though.